toolchain/check/context.cpp

```diff
@@ -225,6 +225,54 @@ auto Context::DiagnoseNameNotFound(SemIRLoc loc, SemIR::NameId name_id)
   emitter_->Emit(loc, NameNotFound, name_id);
 }
 
+// Given an instruction associated with a scope and a `SpecificId` for that
+// scope, forms an instruction that describes the specific scope.
```

Comment:
Minor nit: I intuitively read "forms an instruction" as meaning it creates a new `Inst` object, and maybe allocates an ID for it, rather than returning the ID of an inst that already exists.

```suggestion
// scope, returns an instruction that describes the specific scope.
```